### ISSUE TITLE
flaky tests TestSyncLightWithTrustedPeers and TestSyncLightWithBridge

### ISSUE THREAD

**Issue by liamsi** [celestiaorg/celestia-node#610]
https://gist.github.com/liamsi/8ff1c98dde9ba39160b40124c96ab036

**Comment by renaynay** [comment]
```
2022-04-11T08:51:31.688Z	ERROR	node	node/node.go:97	starting Full Node: listen tcp 0.0.0.0:26658: bind: address already in use
    sync_test.go:262: 
        	Error Trace:	sync_test.go:262
        	Error:      	Received unexpected error:
        	            	node: failed to start: listen tcp 0.0.0.0:26658: bind: address already in use
        	Test:       	TestSyncLightWithTrustedPeers
```

This comes from the fact that an RPC server is started on every node on that exact port. We should make the port configurable so that it can be set randomly for testing + CI

**Comment by Wondertan** [comment]
Closed @renaynay ?

**Comment by adlerjohn** [comment]
Fixed by #617
